<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuHoteles2</class>
 <widget class="QWidget" name="MenuHoteles2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>561</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Agregar</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>481</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="inputCodigoHotel">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Codigo de Hotel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="paisComboBox"/>
      </item>
      <item>
       <widget class="QLineEdit" name="inputNombreHotel">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Nombre Hotel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputTarifaHotel">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Tarifa por Noche/Persona</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputTipoHotel">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Tipo (ej. 5 estrellas)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botonAceptarAgregaHotel">
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modificar</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>371</width>
       <height>292</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="botonBuscarModHoteles">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="inputCodigoHotelMod">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string> Codigo del Hotel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="botonGuardarModHoteles">
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tablaModificarHoteles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>531</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Mostrar">
    <attribute name="title">
     <string>Mostrar</string>
    </attribute>
    <widget class="QTableWidget" name="tablaHoteles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="botonActualizarHoteles">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>410</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Eliminar</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>471</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="inputCodigoHotelEliminar">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Codigo del Hotel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="mesajeRespuesta">
          <property name="placeholderText">
           <string>Aqui se mostrara un Mensaje de Respuesta</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="botonEliminarModHoteles">
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
